Source/SuperLinq/readme.md

```diff
@@ -584,13 +584,27 @@ descending) into a single sequence that preserves that order.
 
 This method has 4 overloads.
 
+### SortedMergeBy
+
+Merges two or more sequences that are in a common order (either ascending or descending)
+according to a key into a single sequence that preserves that order.
+
+This method has 4 overloads.
+
 ### SortedMergeDescending
 
 Merges two or more sequences that are in a common order (either ascending or
 descending) into a single sequence that preserves that order.
 
 This method has 2 overloads.
 
+### SortedMergeByDescending
+
+Merges two or more sequences that are in a common order (either ascending or descending)
```

Comment:
This "either" isn't true for the *Descending methods

Reply:
Corrected